internal/govulncheckshim/client.go

```diff
@@ -0,0 +1,110 @@
+package govulncheckshim
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+
+	"github.com/google/osv-scanner/pkg/models"
+	"golang.org/x/vuln/client"
+	gvcOSV "golang.org/x/vuln/osv"
+)
+
+type localSource struct {
+	vulnList         []models.Vulnerability
+	response         map[string]*models.Vulnerability
```

Comment:
Is there a better name for this? Maybe `byID` or something similar?

Reply:
Good point, renamed all 3 to have better names:
vulnsByID      
vulnsByAlias   
vulnsByModule